bottlecap/src/span_pointers/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod helpers;
```

Comment:
I'd sugges to move `span_pointers` inside `traces`, so `traces/span_pointers/mod.rs` or `traces/span_pointers.rs`

you don't need the `helper.rs`, that's not how rust file structure is recommended.